students/M1lya1/hw-8/task2.js

```diff
@@ -0,0 +1,31 @@
+    
+const itemId = () => {
+    
+    const id = Math.round(Math.random() * 1000);
+    const arrayId = arrayWithObjects.map(item => item.id);
+    // делаем проверку, чтобы не было одинаковых айди
+     if(arrayId.includes(id)){
+        return itemId();
+     
+     } else {return id}
+     
+ };
+ 
+ const arrayWithObjects = [];
+ 
+ const createObject = (...names) => {
+ 
+  [...names].forEach(item => {
```

Comment:
`names` is already an array
```suggestion
  names.forEach(item => {
```